src/Common/MessageContracts/SwitchoverDeployInstance.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace Middleware.Common.MessageContracts;
+
+public record SwitchoverDeployInstance : Message
```

Comment:
Switch over request should be validated by middleware. By this I mean that it should take policies into consideration. A switch over to a cloud with a policy action for always using edge should not active switch over

Reply:
I am fully onboard with the idea. But I believe that first, we should revise how the policies are supposed to work in a separate ticket. When we will have an idea we can start plugging them into the parts of the system